src/utils/scrollParent.js

```diff
@@ -27,7 +27,7 @@ export default (node) => {
       continue;
     }
 
-    if (overflowRegex.test(overflow) && overflowRegex.test(overflowX) && overflowRegex.test(overflowY)) {
+    if (overflowRegex.test(overflow) || overflowRegex.test(overflowX) || overflowRegex.test(overflowY)) {
```

Comment:
'LazyLoad Overflow should not detect an overflow container when only one of the scroll property is auto/scroll FAILED', I don't really get it, users may not want to use all of this 3 properties right?

Reply:
See [here](https://stackoverflow.com/questions/6421966/css-overflow-x-visible-and-overflow-y-hidden-causing-scrollbar-issue/6433475#6433475) for further information about why use `&&` instead of `||`

Reply:
Thank you. That's good to know. However, this condition breaks the example in Chrome. My chrome version is Version 63.0.3239.84 (Official Build) (64-bit).

Reply:
I can try to fix it as well. Thanks for your reply.